cms/static/js/views/video_thumbnail.js

```diff
@@ -216,6 +278,36 @@ define(
                 this.$('.thumbnail-wrapper').attr('class', 'thumbnail-wrapper {action}'.replace('{action}', action));
             },
 
+            validateImageFile: function(imageFile) {
+                var errorMessage = '';
+                var self = this,
+                    fileName,
+                    fileType;
+
+                fileName = imageFile.name;
+                fileType = fileName.substr(fileName.lastIndexOf('.'));
+
+                if (!_.contains(self.getVideoImageSupportedFileFormats().machine, fileType)) {
+                    errorMessage = gettext(
+                        'The selected image contains unsupported image file type. ' +
+                        'Supported file formats are {supportedFileFormats}.'
```

Comment:
This image file type is not supported. Supported file types are .gif, .jpg, and .png.